charts/astronomer/templates/houston/cronjobs/houston-populate-hourly-task-audit-metrics.yaml

```diff
@@ -0,0 +1,53 @@
+########################################
+## Houston Populate Daily Task Metrics CronJob
+########################################
+{{- if .Values.houston.populateHourlyTaskAuditMetrics.enabled }}
```

Comment:
Change this to be based on feature flag instead
`{{- if .Values.global.taskUsageMetricsEnabled }}`

Reply:
Refer this PR - https://github.com/astronomer/astronomer/pull/1682